README.md

```diff
@@ -78,21 +88,23 @@ custom themes.
 
 ## Authors
 
-The main author is Raph Levien.
+The Xi editor project was started by Raph Levien but has since received
```

Comment:
Actually it occurs to me I'm trying to spell this "xi-editor" if for no other reason than to avoid confusion with a certain General Secretary.